### DISCUSSION TITLE
SFTP Destination

### DISCUSSION THREAD

**Discussion by ToniRegidor** [drakkan/sftpgo#1571] [Q&A]
Hello,

I have started with the cloud free-trial to test it, and I'm trying to transfer files from one SFTP into another SFTP.

I was able to configure the source with no problem and it seems to connect, but when trying to set the destination, first I don't find just an SFTP destination, all I can find is SFTP-JSON, not ideal, but enought to test; but when I set all of the data to make the connection (host, username, password, destination path and port) I get the following error: 

_An exception occurred: failure: remove The process cannot access the file because it is being used by another process.. Stacktrace: Traceback (most recent call last): File "/airbyte/integration_code/destination_sftp_json/destination.py", line 76, in check writer.delete(stream)_

This destination SFTP it's in my local machine, and none else it's accessing it, and inside the folder I find a .jsonl file, I guess that's the file gives an error of deletion.

My question is, it's there a way that I can just transfer files from one SFTP to another SFTP using airbyte?

**Comment by drakkan** [comment]
Hello,

I think you are referring to different software, your stack trace refers to Python code. SFTPGo is not written in Python

**Reply by ToniRegidor** [reply]
My apologies.

Your software works great, I had no problems installing and deploying the SFTP.

**Comment by ToniRegidor** [comment]
I mistaken the source.
